### PR TITLE
Adding a link to the legal copy for demographics

### PR DESCRIPTION
**Pull request by staubina**
Screen Shot 2020-07-09 at 10 24 57 AM

Screen Shot 2020-07-09 at 12 44 04 PM

### CODE REVIEW

File: src/account-settings/demographics/DemographicsSection.jsx [hunk 1]
```diff
@@ -24,6 +25,7 @@ import { Input } from '@edx/paragon';
 import PropTypes from 'prop-types';
 import React from 'react';
 import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
```
Comment (Tj-Tracy) [inline src/account-settings/demographics/DemographicsSection.jsx:28]: small nit: remove the import if its not being used. Unless I missed it I think you removed any Link components.

File: src/account-settings/demographics/DemographicsSection.jsx [hunk 2]
```diff
@@ -169,7 +171,12 @@ class DemographicsSection extends React.Component {
         <h2 className="section-heading">
           {this.props.intl.formatMessage(messages['account.settings.section.demographics.information'])}
         </h2>
-        {this.renderDemographicsServiceIssueWarning()}
+        <div className="mb-2">
+          <a href={getConfig().MARKETING_SITE_BASE_URL + '/demographics'} target="_blank">
+            {this.props.intl.formatMessage(messages['account.settings.section.demographics.why'])}
+          </a>
+          {this.renderDemographicsServiceIssueWarning()}
```
Comment (oliviaruizknott) [inline src/account-settings/demographics/DemographicsSection.jsx:178]: Curious: have you seen what this looks like if the warning is visible?

Comment (staubina) [reply src/account-settings/demographics/DemographicsSection.jsx:178]: Added one without, let me get one with the warning.

Comment (staubina) [reply src/account-settings/demographics/DemographicsSection.jsx:178]: @oliviaruizknott I added both screenshots.

**Pull request merged**